README.md

```diff
@@ -92,27 +111,27 @@ function handleSubmit(event) {
 ```
 
 Since we don't have a server to send our data to, let's remove our
-`sendFormDataSomewhere()` function. Instead, we'll demonstrate submission by
-modifying our `Form` component to access submitted values from state and list
-them in the DOM:
+`sendFormDataSomewhere()` function, the `props` parameter, and the `Props`
```

Comment:
Recommend changing "the `props` parameter" to "the `Form` component's `props` parameter"